### ISSUE TITLE
Improve upgrade command status

### ISSUE THREAD

**Issue by regg00** [regg00/ChocoMan#12]
**Is your feature request related to a problem? Please describe.**
Not a problem, but right now, when running `Install-ChocoPackage` with the `-Upgrade` switch, if the package is already up to date, then the message is not clear enough.

**Describe the solution you'd like**
Return a status message stating that the package is already on the latest version.

**Additional context**
Here's an example of an upgrade request on an already upgraded package.
``` powershell
PS C:\Users\regg0> install-ChocoPackage -Name winscp -Upgrade -Verbose
VERBOSE: Chocolatey is installed.
VERBOSE: Chocolatey is installed.
VERBOSE: Command to execute: choco upgrade -y winscp --source chocolatey -r --no-color
VERBOSE: Chocolatey is installed.
VERBOSE: Chocolatey is installed.
VERBOSE: Command to execute: choco list -r --no-color

Name   Status Version
----   ------ -------
winscp        6.1.1
```

**Comment by DailenG** [comment]
Love this project, i started a similar project that checks for updates and caches them in a json file so they can be scheduled to update later but without rechecking for updates. Yours is much better organized and polished. My project is geared more towards supported internal/custom repos. I'm going to try and help contribute when possible and will keep an eye on this repo and open issues.

**Comment by regg00** [comment]
Thanks @DailenG. It's nice to know someone love this project. 
Not a bad idea to be able to schedule the update. I think I'll try to implement that.

**Comment by regg00** [comment]
fixed in #18
